src/include/setup_particles.hxx

```diff
@@ -90,7 +91,6 @@ struct SetupParticles
         pzi *= gam;
       }
     }
-
     return psc::particle::Inject{pos, {pxi, pyi, pzi}, wni, npt.kind, npt.tag};
```

Comment:
These are the kind of changes I'd prefer to avoid, but don't worry about them for now.